<ui version="4.0" >
 <class>StardustConfig</class>
 <widget class="QDialog" name="StardustConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Stardust configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Username:</string>
       </property>
       <property name="buddy" >
        <cstring>m_userName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="m_userName" >
       <property name="maxLength" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Password:</string>
       </property>
       <property name="buddy" >
        <cstring>m_password</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="m_password" >
       <property name="maxLength" >
        <number>20</number>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QCheckBox" name="m_savePassword" >
       <property name="text" >
        <string>&amp;Save password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_automaticallyConnect" >
     <property name="text" >
      <string>&amp;Automatically connect on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_requestConfirmation" >
     <property name="text" >
      <string>&amp;Request confirmation before reporting</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StardustConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>335</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel" >
     <x>363</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StardustConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel" >
     <x>127</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
